data/scenarios/Challenges/Sokoban/foresight.yaml

```diff
@@ -92,7 +92,7 @@ entities:
       char: '#'
     description:
       - Unmovable barrier
-    properties: [known, unwalkable]
+    properties: [known, unwalkable, boundary]
```

Comment:
I think this one will probably look strange with connected walls, won't it?

Reply:
I don't mind the aesthetic, but it does look a bit like random ladders/staircases, which could be a bit misleading.
Maybe at some point I'll consider a more sophisticated renderer that eliminates interior boundaries.

![image](https://github.com/user-attachments/assets/91e4c78b-88a6-423f-94d5-47b08d0e2ef9)

Reply:
It's an interesting challenge to specify how such a more sophisticated renderer should work.  Like, I think I have an intuitive idea of what I think the result should look like (though it may not be the same as yours), but I think there is some interesting graph theory needed to compute the right drawing.  I will see if I can give it some thought.